app/uk/gov/hmrc/agentsubscription/service/SubscriptionService.scala

```diff
@@ -23,6 +23,7 @@ import play.api.mvc.Request
 import uk.gov.hmrc.agentmtdidentifiers.model.{ Arn, Utr }
 import uk.gov.hmrc.agentsubscription._
 import uk.gov.hmrc.agentsubscription.audit.{ AgentSubscriptionEvent, AuditService }
+import uk.gov.hmrc.agentsubscription.auth.AuthActions.AuthIds
```

Comment:
unused import?

Reply:
looks like it was removed in another PR , cant see it anymore